### ISSUE TITLE
RW loss

### ISSUE THREAD

**Issue by qdd1234** [jmlipman/RegionWiseLoss#1]
Hi, When I use RW loss, I find a question that during training process, the loss value is negative, Could you tell me why?

**Comment by jmlipman** [comment]
Hi! The RRW map contains negative values, so, multiplying it with the softmax values (positive values) will lead to some negative values, which ultimately may lead to a negative loss. Nevertheless, this should not affect convergence or optimization stability.
